src/app/app.js

```diff
@@ -16,17 +17,45 @@ import useQueryClient from './use-query-client.js'
 const App = () => {
     const queryClient = useQueryClient()
 
+    // TODO: fetch this instead of hardcoding
+    const currentItem = {
+        id: 'aC392Jk3200',
+        name: 'Malaria treated at PHU without ACT',
+        code: 'DE2398742',
+        lastUpdated: {
+            at: new Date('2022-01-14'),
+            userDisplayName: 'Mwangi Babatunde',
+        },
+        markedForFollowup: false,
+        type: 'numerical',
```

Comment:
This is how I imagined the schema of data items would be based on the requirements in the design spec.